src/ds/d400/d400-motion.cpp

```diff
@@ -131,6 +131,17 @@ namespace librealsense
         }
     }
 
+    ds_motion_sensor & d400_motion::get_motion_sensor()
+    {
+        return dynamic_cast< ds_motion_sensor & >( get_sensor( _motion_module_device_idx.value() ) );
```

Comment:
Can we return some shared ptr and keep it as shared ptr?
What does `get_sensor` return?